src/components/table.js

```diff
@@ -916,6 +916,7 @@ export const ColumnSettings = ({ columnSettings, onChange }) => {
  * @param {Object[]} columnSettings - current column list, in order
  * @param {string} columnSettings[].name
  * @param {bool} columnSettings[].visible
+ * @param modalWidth - Width of the selector modal
```

Comment:
I added this, but I think the format is wrong. Rather than the variable name, it should be the type.

Reply:
I've fixed this and the rest of the params, since they weren't correct either -- since they're destructured parameters you have to use an object prefix for the param name to make the jsdoc work.